packages/features/orgs/SubdomainProvider.tsx

```diff
@@ -0,0 +1,40 @@
+import type { ReactNode } from "react";
+import { createContext, useContext, useEffect, useState } from "react";
+
+import { trpc } from "@calcom/trpc";
+
+interface SubdomainContextType {
+  subdomain: string | undefined;
+  isSubdomain: boolean;
+  logoSrc?: string;
+}
+
+const SubdomainContext = createContext<SubdomainContextType>({ subdomain: undefined, isSubdomain: false });
+
+export const useSubdomainContext = () => {
+  return useContext(SubdomainContext);
+};
+
+interface SubdomainProviderProps {
+  children: ReactNode;
+}
+
+export const SubdomainProvider = ({ children }: SubdomainProviderProps) => {
+  const [enabled, setEnabled] = useState(true);
+  const { data } = trpc.viewer.orgs.getLogo.useQuery(undefined, { enabled });
```

Comment:
We have to disable this query from running after its already been set in the provider. Not need to keep fetching logo URL from endpoint after the inital launch. 

Any subdomain changes would hard refresh the page anyway so we dont need to force this.